example-problems/gradle-project/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
   }
 }
 
-apply plugin: 'com.google.cloud.tools.linkageChecker'
+apply plugin: 'com.google.cloud.tools.linkagechecker'
```

Comment:
Updated to avoid camelCase. This follows Gradle's example https://github.com/gradle/guides/blob/master/subprojects/testing-gradle-plugins/samples/code/url-verifier-plugin/src/functTest/groovy/org/gradle/sample/UrlVerifierPluginFunctionalTest.groovy#L18